volume/store/store.go

```diff
@@ -260,7 +260,10 @@ func (s *VolumeStore) create(name, driverName string, opts, labels map[string]st
 		if v.DriverName() != driverName && driverName != "" && driverName != volume.DefaultDriverName {
 			return nil, errNameConflict
 		}
-		return v, nil
+		// check exist in driver
+		if driverName == "" || driverName == volume.DefaultDriverName {
```

Comment:
Instead of doing this, why not grab the volume driver from the `v` object and do a lookup against the driver?